src/applications/caregivers/definitions/content.js

```diff
@@ -15,6 +15,7 @@ export const links = {
   },
   caregiverHelpPage: {
     link: 'https://www.caregiver.va.gov/',
+    label: 'www.va.caregiver.gov',
```

Comment:
That's not a really nice human-readable label... and instead of using `link` as a key, it might be better to use `url`.

Reply:
That is what the designs called for just running it through this object so I can reference it else where